relations/src/shielder/circuit_utils.rs

```diff
@@ -54,7 +54,7 @@ impl AllocVar<(u8, Result<u64, SynthesisError>), CircuitField> for PathShapeVar
         let ns = cs.into();
         let cs = ns.cs();
 
-        let mut shape = vec![];
+        let mut shape = Vec::new();
```

Comment:
fyi: `vec` macro is available in `core` or `ark_ff` or `ark_std` crates as well

Reply:
This one is from `ark_std` though (or at least that's what I expect).